server/methods/catalog.app-test.js

```diff
@@ -1114,7 +1095,11 @@ describe("core product methods", function () {
         selector: { type: "simple" },
         validate: false
       });
-      expect(() => Meteor.call("products/publishProduct", product._id)).to.not.throw(Meteor.Error, /Access Denied/);
+
+      expect(() =>
+        Meteor.call("products/publishProduct", product._id)
+      ).to.throw(Meteor.Error, /Bad Request/);
```

Comment:
see comment below in `catalog.js`. further, the description suggests this test is more about validation than permission